apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -373,72 +451,41 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
           </>
         )}
       />
-      <SettingsToggle
-        labelClassName="text-sm"
-        toggleSwitchAtTheEnd={true}
-        switchContainerClassName={classNames(
-          "border-subtle rounded-lg border py-6 px-4 sm:px-6",
-          hashedLinkVisible && "rounded-b-none"
-        )}
-        childrenClassName="lg:ml-0"
-        data-testid="hashedLinkCheck"
-        title={t("enable_private_url")}
-        Badge={
-          <a
-            data-testid="hashedLinkCheck-info"
-            target="_blank"
-            rel="noreferrer"
-            href="https://cal.com/docs/core-features/event-types/single-use-private-links">
-            <Icon name="info" className="ml-1.5 h-4 w-4 cursor-pointer" />
-          </a>
-        }
-        {...shouldLockDisableProps("hashedLink")}
```

Comment:
Same here ?

Reply:
Added it back to the singleUseLinks. Missed it. 🙇